packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/tests/index.test.js

```diff
@@ -30,6 +34,14 @@ jest.mock('../../../../../../shared/hooks', () => ({
 
 jest.mock('../SelectedEntriesModal', () => () => <div>SelectedEntriesModal</div>);
 
+const client = new QueryClient({
```

Comment:
Once you have `wrapper` implemented: please move this into the wrapper function, because you don't need the client in the global scope

Reply:
done, thanks